.drone.yml

```diff
@@ -4587,17 +4609,43 @@ steps:
     detach: true
     volumes:
       - name: build
-        path: /artifacts
+        path: /build
       - name: cache
         path: /cache
       - name: tezos-data
-        path: /home/tezos/data
+        path: /data
     environment:
       SODIUM_USE_PKG_CONFIG: 1
     commands:
-      - cp $${BUILD_ARTIFACTS_PATH}/build_files/protocol-runner /
-      - export LD_LIBRARY_PATH="$${BUILD_ARTIFACTS_PATH}/build_files/ffi:$$(rustc --print sysroot)"
-      - $${BUILD_ARTIFACTS_PATH}/build_files/light-node --config-file="$${BUILD_ARTIFACTS_PATH}/build_files/tezedge/tezedge_drone.config" --rpc-port=18732 --tezos-data-dir /home/tezos/data --bootstrap-db-path bootstrap_db --identity-file identity.json --disable-bootstrap-lookup  --peer-thresh-low=0 --peer-thresh-high=5 --network "$${NETWORK}" --protocol-runner /protocol-runner --tezos-context-storage=irmin
+      - OCTEZ_IP=$(perl -e 'use Socket; my $a = gethostbyname("octez-node"); print inet_ntoa($a)')
+      - export LD_LIBRARY_PATH="/build/build_files/ffi:$$(rustc --print sysroot)"
+      - >
```

Comment:
@akoptelov what the... this works for multiline commnand in drone file?

Reply:
Lol, I always wondered why we use such long lines in drone and docker compose files, I thought it was just how it was preferred for some reason.